san/get.py

```diff
@@ -12,9 +12,12 @@ def get(dataset, **kwargs):
     query, slug = parse_dataset(dataset)
     if query in CUSTOM_QUERIES:
         return getattr(san.sanbase_graphql, query)(0, slug, **kwargs)
-
-    gql_query = get_gql_query(0, dataset, **kwargs)
-    gql_query = "{\n" + gql_query + "\n}"
+    if query in V2_METRIC_QUERIES:
+        gql_query = san.sanbase_graphql.get_metric(0, query, slug, **kwargs)
+        gql_query = "{\n" + gql_query + "\n}"
```

Comment:
GQL does not care if you pass `{ query}` or `{\n query \n}`. In this case, we're aiming to reduce the request size, so let's just do `gql_query =  "{" + san.sanbase_graphql.get_metric(0, query, slug, **kwargs) + "}"`